.packit.yaml

```diff
@@ -6,6 +6,9 @@ upstream_project_url: https://github.com/packit/requre
 # we are setting this so we can use packit from requre's dist-git
 # packit can't know what's the upstream name when running from distgit
 upstream_package_name: requre
+# Sync the entiere spec-file, including the changelog, when
+# proposing a new release to Fedora Linux.
+sync_changelog: true
```

Comment:
Don't we want to use the same approach as we do in other projects?

* https://github.com/packit/ogr/blob/129afcf5cc2d09e661478b9d571ea5ae535b7763/.packit.yaml#L9
* https://github.com/packit/packit/blob/8b4f760883d5a586fbf489e9aacb4bd76f4a0fda/.packit.yaml#L14

Based on the architecture meeting on 9/3:

> For now, for the packages we maintain, write a changelog in GitHub that doesn't violate the spec-file format.

Reply:
Wow, I completely forgot about this! I'll update this PR and use the same approach.

Reply:
Thank you.